src/Shared/FileUtilities.cs

```diff
@@ -959,7 +959,12 @@ internal static bool FileOrDirectoryExistsNoThrow(string fullPath, IFileSystem f
         /// </summary>
         internal static bool IsSolutionFilename(string filename)
         {
-            return HasExtension(filename, ".sln");
+            return HasExtension(filename, ".sln") || HasExtension(filename, ".slnf");
```

Comment:
Why does this need to be extended?

Reply:
This is used a number of times to check if the input is a solution rather than, say, a project. Extending this function cleanly brings the solution filter file to SolutionFile.cs where it can be parsed.

Reply:
Update the doc comment with `<remarks>` saying that, please

Reply:
ping on extending comment here